docs/cli.md

```diff
@@ -29,17 +34,303 @@ Common Flags
 | --insecure-skip-tls-verify | Skip SSL certificate validation (will make HTTPS insecure) |
 | --help (-h)                | Display help for specified command |
 
-oc get
------------
+## Basic Commands
+
+### oc
+
+This displays the list of available commands.
+
+### oc login
+
+This retrieves a session token that allows you to act as a user.
+Invoked without args, `oc login` prompts for a username and a password.
+For example, these two invocations are identical:
+
+```bash
+$ oc login
+Username: test
+Password: test
+
+$ oc login -u test -p test
+```
+
+See also [`oc logout`](#oc-logout) and [`oc whoami`](#oc-whoami).
+
+### oc new-project
+
+This creates a new project, with you as the project admin.
+Option `--display-name` specifies the user-facing name of the project.
+Option `--description` specifies its description.
+
+For example:
+
+```
+$ oc new-project web-team-dev \
+  --display-name="Web Team Development" \
+  --description="Development project for the web team."
+```
+
+Note that we use double-quotes around the option arguments.
+
+### oc new-app
+
+This creates a new application in OpenShift with the specified source code, templates, and images.
+It builds up the components of an application using images, templates, or code that has a public repository.
+It looks up images on the local Docker installation (if available), a Docker registry, or an OpenShift image stream.
+If you specify a source code URL, it sets up a build that takes the source code and converts it into an image that can run in a pod.
+Local source must be in a git repository that has a remote repository that the OpenShift instance can see.
+The images will be deployed via a deployment configuration, and a service will be connected to the first public port of the app.
+You may either specify components using the various existing flags or let new-app autodetect what kind of components you have provided.
+
+If you provide source code, you may need to run a build with `oc start-build` after the application is created.
+
+The general form is:
+
+```
+$ oc new-app <component> [options]
```

Comment:
Another (simpler) idea is to xref from `oc new-app` to `oc new-build`, which is what i went w/ in the end.